pkg/deploy/dashboard/deployment_dashboard.go

```diff
@@ -22,7 +23,43 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const CHE_SELF_SIGNED_MOUNT_PATH = "/public-certs/che-self-signed"
+const CHE_CUSTOM_CERTS_MOUNT_PATH = "/public-certs/custom"
+
 func (d *Dashboard) getDashboardDeploymentSpec() (*appsv1.Deployment, error) {
+	var volumes []corev1.Volume
+	var volumeMounts []corev1.VolumeMount
+	var envVars []corev1.EnvVar
+
+	volumes, volumeMounts = d.provisionCustomPublicCA(volumes, volumeMounts)
```

Comment:
What if `deploy.CheAllCACertsConfigMapName` does not exist ?

Reply:
Is it really possible?

So far, on kubernetes/openshift I configu nothing custom and it's there, but empty:
```
> kc get configmap ca-certs-merged -o=yaml
apiVersion: v1
kind: ConfigMap
metadata:
  annotations:
    che.eclipse.org/included-configmaps: ""
  creationTimestamp: "2021-09-16T13:36:10Z"
  labels:
    app.kubernetes.io/component: che
    app.kubernetes.io/instance: che
    app.kubernetes.io/managed-by: che-operator
    app.kubernetes.io/name: che
    app.kubernetes.io/part-of: che.eclipse.org
  name: ca-certs-merged
  namespace: eclipse-che
  ownerReferences:
  - apiVersion: org.eclipse.che/v1
    blockOwnerDeletion: true
    controller: true
    kind: CheCluster
    name: eclipse-che
    uid: 771b1491-c400-49a4-8ee6-2665b4c6f6bd
  resourceVersion: "300040"
  uid: 47316f04-d919-4bf5-82ca-b24d898d1f3f
```

Reply:
As far as I see keycloak, che server mounts it without any check if it exists.
Do you still suggest to make such check before mounting?